WalletWasabi.Tests/IntegrationTests/RegTests.cs

```diff
@@ -741,6 +741,7 @@ public async Task SendTestsAsync()
 		{
 			(string password, RPCClient rpc, Network network, Coordinator coordinator, ServiceConfiguration serviceConfiguration, BitcoinStore bitcoinStore, Backend.Global global) = await InitializeTestEnvironmentAsync(1);
 
+			bitcoinStore.IndexStore.NewFilter += Wallet_NewFilterProcessed;
```

Comment:
The alternative is to make the API of WalletManager even more unreasonable. IMO that's a bad compromise.

Reply:
With the combined function CreateAndStart you cannot use events in WalletService that are important before the start of the wallet because you can't have the WalletService without starting it. This case is the first symptom of this.